mappings/3d3170d07cd2ddfd9be6fc24c36309c1a4ccdf43c3520d0395ceb5f3444f52454d4f4e.json

```diff
@@ -0,0 +1,64 @@
gwUBhA5Ph4ricEQOGPcYgdxArloNA2sYfTEm5sg6bSA4UTlB4z+OYBo52MDCFAtJfQNCfQkRKXvUnhmopn4x+e852jnVOdXiiff+HOK98vlvHXrd/CNIjb57Zzvw8KbjNj6++nkCEG8bYbIxEAAwQGltb5ZewhMAkA+pg8l8kOYgyBou86aQs6K54tOG4d+hMcAkYPqvJdu3rh20uNPFCE4faZBig81hAzBvwWhykkSw/UwpaIXyBmwh/wxaWmKDsegpLjwbhzrA5y9wWj88C/492JkegZtYbhgAzEa/ellgmng3uppPPl7J4EhFsmCOR82+axA1lCTvc7o7oTLPaVGYUQrZs5gJQzh2ME3wQACxvcZ51Rntl2gqJLBVBY+PBMmYdChp91fB8YMgnP9gRGezKGyz7kNDNStI19OMe7Hku33zGDc7CBQ6W3bttU6lrWsulUb93v9fRBIT1i4ceOdcf3Vou3sOIG0Cwc2XV8pvCO5T7TGWIKlRKZylIUWowp1nN9A/qTxvvGJBAg8XTGeizaMBVfXGiFB+caERAN8eB8QyRtDsbE8Cfx9dUW+PZqA+Ttr4elo3+DO2eb4/aJEBSTLe6frIWrmxvg42HhaLsoihmEA4TChH47nK2/NwEoIIgFJDilKxwonJmYNCC4ZZk52Ya+Pe3Ae9QAl/VNKGzQ1FehPgcfJNYnYSAgKLjddAXZQED4PhgkIANg8J0vzcB1l20ofDhAWH+FDwhaWx8UcpjmQdibYQKBiUwfAAKIGEL6wjnROdcHko4j3TP0SC/YgJi281gfSFp2oUPmlqUvxCb6Pf9T0L3Y6nsrFGi/fktaQtf0QCAz1lCKqnVPePZWFrKOjlpPMLDsSzrvE+dYozedMCQmBoknPsbXl8Po9Ma4e7YRio+GYm7Pn+F0fCj+kdgK+1cEY92cpw0wt08Eky2C8eWZ2ohd/io+mRHJ9JHClY7sKw1h2mGDsUQvrvdkq+9FQPYkYMK57jOFOZxmrMFwUQ4MMYbbFzCJUX88w2cIbjedoWWBQCxBUCis+JNlFDaUVThQSGS67EOA8Psm/I6qjj4wpCuWMdOQhiAwFELaMtVuR2uzbA1BQcc4ZzsQyOnSAxbDpS3oWF8vmMPpJNuvYy18yIF0pFo1W7qc61jBhSS7Js2ltfodDxQsfeAFgEFQ+MByx6hUaNA2BxCxgvREH5a9DRhuWx+GD2cEBcVnHwJCoOgXzxBCB8xfPw0PLv4RX11sYMCQ829teBrL+v8Is4b9Dutm/Q4FB0Nw91SoaYqHZ2vg5p4W6DdnjFV4b2Y0fRQ2yAwmLmliDD/N7MnlcK37ocQXll4IUUeW+iZ8ttB2lRWBYSHCMwcIlnS8bVcpEFjfBEFAUweY1jsyRFr6qT4IgkL3a1mHsQXNG/2UWf+EliU2WbYlCCQyTVMIGCyDNEjVT6bsQA6T48whnlN8CzjVA43nUA18KSTYeQKBrqEwwNJ6KHmMjjWwBK7hzGU27hydawAQMMQM/B2lvQ4YCi0CwWb0omldx/YmSCyb4TnaJs2hP7SxNJXA6M/WK1M2Ek6HJGwfjUeXX6ToZKZxtiEeUnxm76yJJf2fQMGRRnh4oSFBUR+PztZB8qHWmLx8JLqspHAl6zgwcJnXM6ZgGU6m8HWEiUiCz0zMIPMAIIZQX4RSU4HIB4aA4C/7gPAZwcKGOd03xwRyuDqkBAanGVzHlNJQS0V9gekDQ8yw3A8Znn7gtvYr1nC5HAgCxqcESBtlH875NDmYzpHznHkAkfPY6v39vrkBMi4rbTRGcNcQYJw2cU7TtXz94BhAINKyA6LTD+XgcMBxoBBQ3LE6h/ekUCGn6xiea+yj+7JruXBiYYmAGUDG6C99IWAkEBjUHL2oM5ZvnYis0+8SGI3x1aX61A21cWXJE/jyXD18eaE+ik62woFd7TB21Rw6U/0gDEUJLhz1IlOICfQ/8FYyw7CswsxlIgJFwOk0AUOMoT4JYwcPCBUZwlhCRkDYPg8cPiC6MmRoAEuAsPUK+y1seGaA4TZfTzjQaFmdVtIarm8ioCcUOrj+2ZLVFjY+IzAsfLiua9fyZdY3oYoOOIglLQAKltqubdaPoG2eQ/1+CNMnXBdwDGByltZt2TlV1K99Ot83/xhr9SY63bkq7fqBY7jNtivkCCAChLISsYVAwWxm7XoMZDjpT8YYwFIgEXvoLzbHr1+FrbvH4vLR9rh6+G+IjW6LI/t7YNeefpi2fjbBsxbd6WCJyL4ElEKRWKJnbIKln+pqD7f00vU/WIjwwOGHhQBTSEd4gJD57OCDwQePgcIDRqD0jawg80FSERgGBJq2VWQICxPSEQYMgoLrTmTGOx1BdhAw2oolAqEjBp8uZfiw1NGv9AqgkJPMaRUdwu0DCABLY7lu4KEZoNSy2UIVSux8j2kCpmvpGv66OdNt98FhczDkcLPvAqLifdi1tM79+ksrhRKnJ7zQo/AhYNAsGyFABAiJ0H603gRdL4rTAdw2bC21Apetq1p6gA5XB1g409teBIHA0IdsIxEpQan+CKWjvQgKHwhybEVQOGD4ocIDiZxPZ5cDwwszazxmoIi1rIP7LIzQyT5wfCD4LV/LPhgqgiMACI8lLHR44FDpawp1Vlk48cAgfdGBy22XMITQymdeKVOoWOn+dpbmdM8GrBMwmG1YSNHxcpwnAhkyfH1R8RpyrjGGv03GZQHLHK/jDQBiFB8Mbtl6QnWcjqEJtLpmQODS8cpY3Gwv9WxqnUCxc9VDyvBB5/dZxxbPMKBw0o9lX2YqvQik3jqebNDLQgS3S0Cqr0PG9T5xCdQRBES8HK9/XFTmIQA4dgiPd3+3JJOuMGYQuMgoPfQ2lgGDzqXjfPD42YcxicQnndx1DQWihso9EPjO9s0Hgg8QAwEd74Ojk8cKmkBjmYb2kSE6UWQaY9DEDHK+3kc15tA6TcBoL8FJpmi7NAb/D+w3q1q2eimvAAAAAElFTkSuQmCC",
```

Comment:
Hi @oanhoanhfu89,

please, change the logo as it can lead to IP issues.

Thanks!